2023-03-01 13:59:01,013 [MainThread  ] [INFO ] [root]  STARTING BROWSER: /home/jake/chromium/src/out/handles_full_gc/chrome
2023-03-01 13:59:01,013 [MainThread  ] [INFO ] [root]  STARTING BROWSER: driver: /home/jake/chromium/src/out/handles_full_gc/chromedriver
2023-03-01 13:59:01,013 [MainThread  ] [INFO ] [root]  STARTING BROWSER: args: --no-default-browser-check --disable-component-update --disable-sync --no-experiments --enable-benchmarking --disable-extensions --no-first-run --disable-background-timer-throttling --disable-renderer-backgrounding --no-sandbox --window-size=1500,1000 --enable-logging --log-file=/home/jake/handles_numbers/no_compact_with_stack_30_iters/chrome_v112_chrome-handles-no-compact_no_sandbox_js_flags_no_compact_with_stack/speedometer_2.1/21/browser.chrome.log --js-flags=--no-compact-with-stack
2023-03-01 13:59:04,902 [MainThread  ] [INFO ] [root]  RUNNING STORY
2023-03-01 14:01:15,643 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff1858847c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/bda5fe2b66f7bfe6236ee2881a3bbc28
2023-03-01 14:01:15,644 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff185884cd0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/bda5fe2b66f7bfe6236ee2881a3bbc28
2023-03-01 14:01:15,644 [MainThread  ] [WARNI] [urllib3.connectionpool]  Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff185884a00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/bda5fe2b66f7bfe6236ee2881a3bbc28
